src/pymax/mixins/message.py

```
@@ -265,7 +360,25 @@ async def send_message(
         use_queue: bool = False,
     ) -> Message | None:
         """
-        Отправляет сообщение в чат.
+        Отправляет текстовое сообщение в чат с опциональными вложениями.
```

Ruff (0.14.8)

Docstring contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF002)